.golangci.yaml

```diff
@@ -40,3 +40,8 @@ issues:
         - revive
       # Config Sync uses errors for human-readable status messages.
       text: "error strings should not be capitalized"
+    # ResourceGroup apis currently don't conform to linter
```

Comment:
:'(

Reply:
I wanted to update this before moving it over but it was hard to make changes without them being reflected in the CRD. I figure we re-enable linting after everything is imported so that we can easily see how changes to the comments impact the CRD

Reply:
as long as you plan to do that next, i'm ok with this for now.